doc/manipulating_images/manipulating_images.rst

```diff
@@ -198,7 +198,7 @@ brain. It is thus convenient to apply a brain mask in order to convert the
     :width: 100%
 
 Note that in an analysis pipeline, this operation is best done using the
-:ref:`masker objects <masker_objects>`. For completness, we give code to
+:ref:`masker objects <masker_objects>`. For completeness, we give code to
```

Comment:
...we give the code to do it manually, below...

Reply:
Can do!